test/integration-test-main.sh

```diff
@@ -352,6 +352,30 @@ function test_multipart_copy {
     rm_test_file "${BIG_FILE}-copy"
 }
 
+function test_multipart_mix {
+    describe "Testing multi-part mix ..."
+
+    if [ `uname` = "Darwin" ]; then
+       cat /dev/null > $BIG_FILE
+    fi
+    dd if=/dev/urandom of="/tmp/${BIG_FILE}" bs=$BIG_FILE_LENGTH seek=0 count=1
+    dd if="/tmp/${BIG_FILE}" of="${BIG_FILE}" bs=$BIG_FILE_LENGTH seek=0 count=1
+
+    # modify directly(seek 7.5MB offset)
+    echo -n "0123456789ABCDEF" | dd of="${BIG_FILE}" bs=1 seek=7864320 conv=notrunc
+    echo -n "0123456789ABCDEF" | dd of="/tmp/${BIG_FILE}" bs=1 seek=7864320 conv=notrunc
```

Comment:
This issues 16 1-byte writes -- do you intend this?

Reply:
Yes, I intend to issue 16 1-byte writes.
Would you like to write one 16 byte or something else?

Reply:
I think either is fine, but the test case below needs more than one byte to trigger a failure for some reason.